### PR TITLE
Don't create Helm charts for riff releases

### LINKED ISSUE #123: Remove riff charts, just produce release YAML
Since we are no longer using Helm, we should remove the Helm charts that we produce.

- remove any packaging and publishing of riff charts

- upload installation YAML files to a new location, maybe `https://storage.googleapis.com/projectriff/release/`

- rename `uncharted` directory, maybe call it `target`

- update README as needed

### PR DESCRIPTION
**Pull request by trisberg**
- modify scripts to produce release YAML only

- rename uncharted dir

- only create temporary Helm charts when source is a chart

- change location for release artifacts to projectriff/releases

- modify CI scripts to only use kapp

Fixes #123

### CODE REVIEW

File: .github/workflows/install-riff.sh [hunk 1]
```diff
@@ -12,21 +12,21 @@ readonly slug=${version}-${git_timestamp}-${git_sha:0:16}
 source $FATS_DIR/.configure.sh
 
 if [ ${1:-unknown} = staged ] ; then
-  echo "Using staged uncharts"
-  unchart_base=https://storage.googleapis.com/projectriff/charts/uncharted/snapshots/${slug}
+  echo "Using staged releases"
+  release_base=https://storage.googleapis.com/projectriff/releases/snapshots/${slug}
```
Comment (scothis) [inline .github/workflows/install-riff.sh]:
```suggestion
  release_base=https://storage.googleapis.com/projectriff/release/snapshots/${slug}
```

File: .github/workflows/publish.sh [hunk 2]
```diff
@@ -21,5 +21,5 @@ helm repo index repository/ --url https://projectriff.storage.googleapis.com/cha
 gsutil -h 'Cache-Control: public, max-age=60' cp -a public-read repository/*.tgz gs://projectriff/charts/releases/
 gsutil -h 'Cache-Control: public, max-age=60' cp -a public-read repository/index.yaml gs://projectriff/charts/releases/
 
-# publish uncharts
-gsutil -h 'Cache-Control: public, max-age=60' cp -a public-read gs://projectriff/charts/uncharted/snapshots/${slug}/*.yaml gs://projectriff/charts/uncharted/${version}/
+# publish releases
+gsutil -h 'Cache-Control: public, max-age=60' cp -a public-read gs://projectriff/releases/snapshots/${slug}/*.yaml gs://projectriff/releases/${version}/
```
Comment (scothis) [inline .github/workflows/publish.sh]:
```suggestion
gsutil -h 'Cache-Control: public, max-age=60' cp -a public-read gs://projectriff/release/snapshots/${slug}/*.yaml gs://projectriff/release/${version}/
```

File: .github/workflows/stage.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+F#!/bin/bash
```
Comment (scothis) [inline .github/workflows/stage.sh]:
```suggestion
#!/bin/bash
```

File: .github/workflows/stage.sh [hunk 4]
```diff
@@ -16,5 +16,5 @@ make clean package
 for f in repository/*.tgz; do mv $f $(echo $f | sed s/${version}/${slug}/); done
 gsutil cp -a public-read repository/*.tgz gs://projectriff/charts/snapshots/
 
-# upload uncharts
-gsutil cp -a public-read uncharted/*.yaml gs://projectriff/charts/uncharted/snapshots/${slug}/
+# upload releases
+gsutil cp -a public-read target/*.yaml gs://projectriff/releases/snapshots/${slug}/
```
Comment (scothis) [inline .github/workflows/stage.sh]:
```suggestion
gsutil cp -a public-read target/*.yaml gs://projectriff/release/snapshots/${slug}/
```

File: .github/workflows/stage.sh [hunk 5]
```diff
@@ -16,5 +16,5 @@ make clean package
 for f in repository/*.tgz; do mv $f $(echo $f | sed s/${version}/${slug}/); done
 gsutil cp -a public-read repository/*.tgz gs://projectriff/charts/snapshots/
 
-# upload uncharts
-gsutil cp -a public-read uncharted/*.yaml gs://projectriff/charts/uncharted/snapshots/${slug}/
+# upload releases
```
Comment (scothis) [inline .github/workflows/stage.sh]: is there a reason to still upload the intermediate charts for istio and kafka? Lines 15-17

Comment (scothis) [reply .github/workflows/stage.sh]: Same question with the release script

Comment (trisberg) [reply .github/workflows/stage.sh]: No, should remove that

**Pull request merged**